maven-core/src/main/java/org/apache/maven/plugin/internal/DefaultPluginValidationManager.java

```diff
@@ -23,17 +23,7 @@
 
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.EnumSet;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
asterixs should not be used in imports

Reply:
IIRC, they are automatically added if importing more than 10 (?) classes from the same package

Reply:
spotless and impsort plugins can't fix star -> individual imports because that would require access to the full class path and they really only operate on a single file.
That also mean the opposite (so I was wrong), i.e. they won't refactor multiple imports into a single one I think.  It must have been the IDE.  
Which means the PR should not contain the star. 

@slawekjaranowski it seems the AvoidStarImport is [no more enforced](https://github.com/apache/maven-shared-resources/blob/maven-shared-resources-5/src/main/resources/config/maven_checks_nocodestyle.xml).  I think we removed it based on the assumption that it was handled by spotless/impsort, but that does not seem to be the case

@yuehcw can you remove the star import ?

Reply:
issue added https://issues.apache.org/jira/browse/MSHARED-1401

Reply:
Hello,

I have made the requested changes and pushed the new commit to this pull request. Please review the updates at your earliest convenience. Thank you for your feedback and guidance!